packages/scheduler/src/Scheduler.js

```diff
@@ -533,13 +533,14 @@ if (typeof window !== 'undefined' && window._schedMock) {
   };
 
   // We use the postMessage trick to defer idle work until after the repaint.
+  var port = null;
   var messageKey =
```

Comment:
The overhead of this is no longer necessary in this mechanism since it's a unique channel.